data-contracts/README.md

```diff
@@ -0,0 +1,130 @@
+# Managing Data Contracts in Confluent Cloud
+
+Data contracts consist not only of the schemas to define the structure of events, but also rulesets allowing for more fine-grained validations,
+controls, and discovery. In this demo, we'll evolve a schema by adding migration rules.
+
+I'd like to credit my colleague Gilles Philippart's work as the basis for the schemas and rules in this demo. My goal here is to add examples with commonly used
+developer tools and practices to his work in this area, which can be found [here in GitHub](https://github.com/gphilipp/migration-rules-demo).
+
+## Running the Example
+
+<img src="./images/overview.png" width="1000" height="600">
+
+In the workflow above, we see these tools in action:
+* The [Confluent Terraform Provider](https://registry.terraform.io/providers/confluentinc/confluent/latest/docs) is used to define Confluent Cloud assets (Kafka cluster(s), Data Governance, Kafka Topics, and Schema Configurations).
+* Using the newly created Schema Registry, data engineers and architects define the schema of the events that comprise the organization's canonical data model - i.e. entities, events, and commands that are shared across applications. - along with other parts of the data contract. This includes data quality rules, metadata, and migration rules. A gradle plugin is utilized to register the schemas and related elements of the data contract with the Schema Registry.
+* Applications which producer and/or consume these event types can download the schemas from the Schema Registry. In our example, this is a JVM application built using Gradle. A gradle plugin is used to download the schemas, after which another gradle plugin is used to generate Java classes from those schemas - thus providing the application with compile-time type safety.
+
+### Prerequisites
+
+Clone the `confluentinc/demo-scene` GitHub repository (if you haven't already) and navigate to the `demo-scene` directory:
+
+```shell
+git clone git@github.com:confluentinc/demo-scene.git
+cd demo-scene
+```
+
+Here are the tools needed to run this tutorial:
+* [Confluent Cloud](http://confluent.cloud)
+* [Confluent CLI](https://docs.confluent.io/confluent-cli/current/install.html)
+* [Terraform](https://developer.hashicorp.com/terraform/install?product_intent=terraform)
+* [jq](https://jqlang.github.io/jq/)
+* [Gradle](https://gradle.org/install/)
```

Comment:
since there's no gradle wrapper, document a specific gradle version or range? or should this work for most versions that folks might have?

Reply:
The setup script actually creates the gradle wrapper. But you're correct, I'll add a specific base version to the prerequisites.